modules/core/core-api/src/main/java/com/enonic/xp/api/ApiDescriptor.java

```diff
@@ -62,15 +62,14 @@ public String getDocumentationUrl()
         return documentationUrl;
     }
 
-    public Boolean isSlashApi()
+    public boolean isMount()
     {
-        return slashApi;
+        return mount;
     }
 
     public boolean isAccessAllowed( final PrincipalKeys principalKeys )
     {
-        return allowedPrincipals == null || principalKeys.contains( RoleKeys.ADMIN ) ||
-            allowedPrincipals.stream().anyMatch( principalKeys::contains );
+        return allowedPrincipals.stream().anyMatch( principalKeys::contains );
```

Comment:
admin role should always have access, regardless of what descriptor says

Reply:
Good catch